### ISSUE TITLE
use gromologist?

### ISSUE THREAD

**Issue by orbeckst** [Becksteinlab/GromacsWrapper#218]
https://gitlab.com/KomBioMol/gromologist is a Python package for handling top and itp files.

**Comment by orbeckst** [comment]
Or rather just remove ALL handling of TOP and ITP files and just leave a note for people to look at gromologist.

**Comment by jandom** [comment]
I've only had a cursory look at Gromologist but i'm pretty impressed: the documentation seems really extensive, and there are tests (!!). Academic code has gotten better lately :D 

For example, check out this bit [about REST2](https://gitlab.com/KomBioMol/gromologist#preparing-rest2-topologies) – it would have saved me weeks of awk scripting during my PhD... 

Putting on my startup hat: we can choose to invest time in these features and try to catch-up but it'd be a considerable investment. The typical strategy is to "pivot" into an area where we have an advantage: idk what that would be, maybe ability to run gromacs from python?

**Comment by orbeckst** [comment]
Given limited resources I'd rather specialize on the "provide a working Python-like interface to GROMACS tools that may allow you to run the same script with different versions of the GROMACS tools."

**Comment by jandom** [comment]
Agreed – what could change this decision? Perhaps if it turns out that Gromologist is riddled with bugs, or otherwise hard to integrate with (has monster dependencies)

**Comment by orbeckst** [comment]
Yes. Or if deps are an issue, make deps optional. 

1. Would you want to rewrite [gromacs.scaling](https://gromacswrapper.readthedocs.io/en/latest/gromacs/blocks/scaling.html#gromacs-scaling-partial-tempering) to use gromologist or is that superfluous because their website already shows an example of how to do this kind of stuff directly?

2. Given that gromologist, exists would we want to maintain the existing ITP/TOP code and the scaling module for as long as it's feasible (i.e., leave it as long as it does not create problems, possibly add a deprecation warning) or just rip it out before a 1.0 (which would require a 0.9.0 release with deprecation warnings).

**Comment by jandom** [comment]
I just looked at [gromacs.scaling](https://gromacswrapper.readthedocs.io/en/latest/gromacs/blocks/scaling.html#gromacs-scaling-partial-tempering) and I swear I have no memory of writing that :D

Not sure what's the best 1st step - could try to re-write that and see how easy that is. I think some scouting work (by me) is needed to inform the decision. 

I would probably rip out, knowing what I know today, because it's less work and it's not clear how many people depend on this (probably not many).
